external/SwiftArgumentParser/BUILD

```diff
@@ -0,0 +1,22 @@
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+swift_library(
+    name = "ArgumentParserToolInfo",
+    module_name = "ArgumentParserToolInfo",
+    srcs = glob([
+      "Sources/ArgumentParserToolInfo/**/*.swift"
+    ], allow_empty = False),
+    visibility = ["//visibility:private"],
+)
+
+swift_library(
+    name = "ArgumentParser",
+    module_name = "ArgumentParser",
+    srcs = glob([
+      "Sources/ArgumentParser/**/*.swift"
+    ], allow_empty = False),
+    visibility = ["//visibility:public"],
```

Comment:
Same here regarding visibility.

Reply:
This is needed to ensure that this external dependency is visible to the repository dependencies, its a quirk of Bazel

Reply:
Same here?

```suggestion
    visibility = ["//SourceKittenLib:__subpackages__""],
```